index.html

```diff
@@ -20,27 +22,160 @@
+            <a href="#" class="date-category">2015</a>
+          </div>
+        </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="hashtag-card">
+          <a href="#" class="hashtag">html</a>
+          <a href="#" class="hashtag">css</a>
+          <a href="#" class="hashtag">javaScript</a>
+        </div>
+        <button type="button" class="see-project">See Project</button>
+      </article>
+
+      <article class="card-4">
+        <div class="card-content"></div>
+        <img src="./images/multipost2.png" alt="Multi-Post Project" class="project-image">
+        <div class="card-headers">
+          <h2><a href="#" class="project-title">Multi-Post Project</a></h2>
+          <div class="card-categories">
+            <a href="#" class="main-category">Canopy</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="sec-category">Back End Dev</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="date-category">2015</a>
+          </div>
+        </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="hashtag-card">
+          <a href="#" class="hashtag">html</a>
+          <a href="#" class="hashtag">css</a>
+          <a href="#" class="hashtag">javaScript</a>
+        </div>
+        <button type="button" class="see-project">See Project</button>
+      </article>
+    </div>
+    <div class="about">
       <h1 class="title">
-        I'm Luis <br>Glad to see you!
+        About Myself
       </h1>
-      <p class="descr">I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      <p class="descr">Hello! I’m a software developer! I can help you build a product, feature or website. Look through
+        some
+        of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to
+        contact me.</p>
       <h2 class="connect"> <a href="#">Let's connect</a></h2>
       <ul class="socials">
         <li><a href="https://twitter.com/lujimhe" target="_blank" rel="noopener noreferrer">
-          <img src="./resources/TwitterVector.svg" alt="Twitter logo" ></a></li>
+            <img src="./resources/TwitterVector.svg" alt="Twitter logo"></a></li>
         <li><a href="https://www.linkedin.com/in/lujim/" target="_blank" rel="noopener noreferrer">
-          <img src="./resources/LinkedinVector.svg" alt="Linkedin logo"></a></li>
+            <img src="./resources/LinkedinVector.svg" alt="Linkedin logo"></a></li>
         <li><a href="https://medium.com/@lu.jim" target="_blank" rel="noopener noreferrer">
```

Comment:
- [ ] Kindly fix the styling in the about section for mobile version.
![Captura de pantalla_2021-09-19_00-28-00](https://user-images.githubusercontent.com/53111464/133916540-eb7a9785-6e43-4d6b-914d-558888d8b907.png)
![Captura de pantalla_2021-09-19_00-29-19](https://user-images.githubusercontent.com/53111464/133916561-f5ea32fe-5495-4de8-9cec-e143bb80c12b.png)